### DISCUSSION TITLE
Task Management

### DISCUSSION THREAD

**Discussion by isle9** [mgmeyers/obsidian-kanban#209] [Ideas & Feature Requests]
Any plans into making this plugin into a fully-fledged task management plugin? Compared to other attempts at making a task plugin for Obsidian, you already have a great GUI, while most of them have none or minimal graphical capabilities.

**Comment by mgmeyers** [comment]
Hey @isle99, could you expand on what you mean by fully-fledged task management? What features would this require?

**Reply by isle9** [reply]
Probably the most important and basic feature would be a way to filter tasks. You could make a GUI in which you could filter tasks by date, projects, tags, etc.

You can check out the new tq-obsidian plugin on how they are doing it.

**Comment by JamesAlastair** [comment]
Hi Mathew, Great Job on this Plugin, thanks! Following @isle99's Task Management feature request, I was looking for a way to create a "Kanban Dashboard", where all the Cards (tasks) in a particular Column (e.g. ToDo Today Column) in all my various Obsidian Kanban Boards show up in a single dashboard view. I guess being able to auto-tag every Note that gets dragged to a designated column in each Board would be one workaround. But even forgetting automation, I can't see a way to manually add a Tag to individual Cards. Can you think of a way to enable that in a future update?
